src/pybind/mgr/cephadm/tests/test_cephadm.py

```diff
@@ -1,5 +1,9 @@
 import json
-from unittest.mock import AsyncMock
+try:
+    # AsyncMock was not added until python 3.8
+    from unittest.mock import AsyncMock
+except ImportError:
+    pass
```

Comment:
```suggestion
    AsyncMock = None
```
?

Reply:
or maybe we should install asyncmock from pip via requirements.txt?

something like:
https://github.com/mgfritch/ceph/commits/async-mock-py3-8

Reply:
@mgfritch I like this and it worked on my py 3.7 environment. I brought the sample commit you had there to install asyncmock with pip into this PR. Could you explain what this other commit you have is about?

https://github.com/mgfritch/ceph/commit/5db2533d4a1092a3bc870ffe29f29b214bf79d7c

Is it necessary when installing with pip or is it just an unrelated improvement? I didn't need it for the test to pass locally using py 3.7.

Reply:
The additional commit relates to 9b334fe97197 where `asyncio.run` and `AsyncMock` were introduced.

I was testing against py36 which needs an event loop impl. since `asyncio.run` does not exist until py37 or later.

Reply:
Okay, cool. I'm seeing the same when testing with py 3.6. Adding that other commit here as well, thanks for figuring this out.